src/components/cart/cart.tsx

```diff
@@ -23,7 +23,7 @@ export const Cart = ({ className, initialIsOpen }: CartProps) => {
         if (initialIsOpen !== undefined) {
             setIsOpen(initialIsOpen);
         }
-    }, [setIsOpen]);
```

Comment:
now component is broken... because prop is no longer `initialIsOpen`, it's now just `isOpen`.